packages/container/tests/owner_test.js

```diff
@@ -1,16 +1,17 @@
 import { getOwner, setOwner, OWNER } from 'ember-utils';
+import { moduleFor, AbstractTestCase } from 'internal-test-helpers';
 
-QUnit.module('Owner', {});
+moduleFor('Owner', class extends AbstractTestCase {
+  ['@test An owner can be set with `setOwner` and retrieved with `getOwner`']() {
```

Comment:
I'd generally prefer to add `assert` as the callback arg and use it below (instead of `this.assert`).  It is simpler, and "feels" more idiomatic to me.

@chancancode / @thoov / @locks - Thoughts?

Reply:
I've been doing `this.assert` because of the code linked in the quest issue, but I prefer the argument version too.

Reply:
I agree with having assert an argument over off of `this`. Definitely is more "standard" when looking at docs and existing testing approaches in the wild. I will update the issue to have more clear explanations about things we are looking for and better examples